src/style.css

```diff
@@ -74,18 +77,42 @@ h6 {
 }
 
 .title {
-  font-size: 40px;
-  margin-left: 40px;
+  font-size: 80px;
   margin-top: 50px;
+  text-align: center;
+  font-family: 'Keania One', cursive;
+  background: linear-gradient(-45deg, #ee7752, #e73c7e, #23a6d5, #23d5ab);
+  background-size: 400% 400%;
+  animation: gradient 9s ease infinite;
+  -webkit-background-clip: text;
+  -moz-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  -moz-text-fill-color: transparent;
+}
+
```

Comment:
- Kindly make sure You use the appropriate color and disposition for the title it will make your design much more to the design provided as it was required in the project :+1:

-  The refresh button must have the same style as it shown in the design provided, I suggest you to change the color text and the color of it border to black instead of green :+1: 

## your design 
![image](https://user-images.githubusercontent.com/95297251/204088219-047c307d-0ed3-4a89-8517-7cac4ca1909d.png)


## Design provides 

![image](https://user-images.githubusercontent.com/95297251/204088166-49b6af57-34fe-4d9c-bdaa-5d99ffcefcf5.png)

Reply:
hey @MaganaAsiati . Thank you for your feedback.
From the requirements.. 

Now you should improve the look and feel of the application, adding the styles of your choice.
Please keep the general layout of the wireframe, this is the only mandatory requirement.

I believe i've followed this... the layout is the same.But the styles are of my choice.

Reply:
@MaganaAsiati 
I believe layout here means the structure... but things like color font stylings are left to my discretion